projects/plugins/jetpack/_inc/client/recommendations/constants.js

```diff
@@ -9,7 +9,7 @@ export const RECOMMENDATION_WIZARD_STEP = {
 	CREATIVE_MAIL: 'creative-mail',
 	SITE_ACCELERATOR: 'site-accelerator',
 	PUBLICIZE: 'publicize',
-	SECURITY_PLAN: 'security-plan',
+	SECURITY_PLAN: 'protect',
```

Comment:
I would have thought the point of having constants like this is so that if we have to change the value, we don't also have to go change everywhere where the constant is used. Is there another reason for it that I am unaware of?